storm-core/src/ui/public/index.html

```diff
@@ -143,6 +143,12 @@
             ]
           });
           $('#topology-summary [data-toggle="tooltip"]').tooltip();
+          $.getJSON("/api/v1/cluster/configuration", function(json){
+              var displayResource = json["scheduler.display.resource"];
+              if (!displayResource){
+                  $('#topology-summary td:nth-child(10),#topology-summary th:nth-child(10)').hide();
```

Comment:
I tried to pass "schedulerDisplayResource" to each supervisor-summary and topology-summary in REST and then use this to check whether we want to display in template htmls. However, it seems a little weird to store a cluster-wide configuration as a supervisor-specific or topology-specific field. Also, when there is no topology submitted, it is not feasible to check whether we want to display the assignedCPU in the topology-summary table's header (similarly for supervisor summary table).

We may rather keep it still since later we will have RAS as default scheduler and will merge with JSTORM's UI features.